src/main/java/sim/coffee/OrderItem.java

```diff
@@ -1,5 +1,15 @@
 package sim.coffee;
 
+import java.math.BigDecimal;
+
 public abstract class OrderItem {
 
+    protected MenuItem menuItem;
```

Comment:
Think this should probably be private
```suggestion
    private MenuItem menuItem;
```